api/v1/installation_types.go

```diff
@@ -489,6 +489,11 @@ type IPPool struct {
 	// Default: 26 (IPv4), 122 (IPv6)
 	// +optional
 	BlockSize *int32 `json:"blockSize,omitempty"`
+
+	// DisableBGPExport specifies whether routes from this IP pool's CIDR are exported over BGP
+	// Default: false
```

Comment:
Are we missing defaulting for this field? The comment claims this defaults to false but I don't see the code for that. Could probably work with a kubebuilder default tag.

Reply:
I've added kubebuilder tag to do the defaulting as suggested 👍